test/simplifyExpression/basicsSearch/basicTest.js

```diff
@@ -21,25 +24,20 @@ describe('basic rules', function() {
         // TODO: handle this case better
         //['3x^2 * 5', '5 (3 x^2)']
     ];
-    tests.forEach(t => testSimplify(t[0], t[1], basic_rules.REARRANGE_COEFF));
```

Comment:
as a side note (since you've already changed it) use camel case and not snake case for naming stuffs